src/views/Viewer.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="viewer">
+    <h1>Dit is de over Viewer pagina</h1>
+    <liwo-map />
+  </div>
+</template>
+
+<script>
+import LiwoMap from '../components/Map.vue'
```

Comment:
I'd suggest renaming the map component to liwo-map so that the webcomponent element and file share the same name